### ISSUE TITLE
Media library: Delete individual search requests

### ISSUE THREAD

**Issue by WebworkrNet** [mediathekview/zapp#259]
### Feature Request:
Possibility to individually delete historical search terms

**Comment by WebworkrNet** [comment]
According to the changelog, search suggestions have been switched off with version 6.0.0.

I notice that new search requests are no longer saved and past ones are no longer available. Is there a connection to the deactivation?

**Comment by cemrich** [comment]
Yes, this ticket is not needed in Zapps current condition. But a new filter saving system is planned - so I'll leave this open for now.
